tools/mo/automation/package_BOM.txt

```diff
@@ -41,12 +41,10 @@ openvino/tools/mo/back/MatMulNormalizer.py
 openvino/tools/mo/back/MaxPool.py
```

Comment:
why do we need this file? is not it outdated?

Reply:
This file is a part of BOM test. BOM test checks license at the head of each code file. So it looks like we still need it.